src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -1,45 +1,65 @@
 #include "z_boss_ganon2.h"
+#include "overlays/actors/ovl_Demo_Gj/z_demo_gj.h"
+#include "overlays/actors/ovl_En_Zl3/z_en_zl3.h"
+#include "objects/object_ganon/object_ganon.h"
+#include "objects/object_ganon2/object_ganon2.h"
+#include "objects/object_ganon_anime3/object_ganon_anime3.h"
 
 #define FLAGS 0x00000035
 
 #define THIS ((BossGanon2*)thisx)
 
+typedef struct {
+    /* 0x00 */ u8 type;
+    /* 0x01 */ u8 unk_01;
```

Comment:
```suggestion
```
This is probably padding and can be removed.

Reply:
It's used: https://github.com/zeldaret/oot/blob/b93dafcada89c36befca3b11b4d05828762d25bd/src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c#L467.